WORKSPACE

```diff
@@ -154,83 +165,78 @@ haskell_cabal_binary(name = "happy", srcs = glob(["**"]), visibility = ["//visib
 
 load(
     "//:build/common.bzl",
-    "tree_sitter_node_types_git",
-    "tree_sitter_node_types_release",
+    "tree_sitter_node_types_hackage",
 )
 
-tree_sitter_node_types_release(
+tree_sitter_node_types_hackage(
+    name = "tree-sitter-go",
+    sha256 = "364a0ae4e683bda1e348fa85c6828cad72122af155560b680f6052852d98db6c",
+    version = "0.5.0.1",
+)
+
+tree_sitter_node_types_hackage(
     name = "tree-sitter-python",
-    sha256 = "50d3fa560391dc4ab8d9a3466f68f2c6a4c12f9cc6421358d2c307023bd740ab",
-    version = "0.16.0",
+    sha256 = "36aca4989a9f8b52d6af1586e6eecc8c3a8db2b5643f64ef13ab3d284c266522",
+    version = "0.9.0.2",
 )
```

Comment:
Rather than pinning against the languages’ releases on GH, we download them directly from Hackage. In the future, if we drop `cabal` support, we could depend on the `haskell-tree-sitter` repository directly, but this seems fine for now.